tests/integration/assessment/test_assessment.py

```diff
@@ -123,14 +129,53 @@ def test_spn_crawler_no_config(ws, inventory_schema, make_job, make_pipeline, sq
     spn_crawler.snapshot()
 
 
+def test_spn_crawler_deleted_cluster_policy(
```

Comment:
Integration tests have to be in the "tests/integration" folder, not the "tests/unit", otherwise they are not run